app/models/deploy.rb

```diff
@@ -127,27 +127,23 @@ def self.expired
   end
 
   def csv_buddy
-    if not (stage.deploy_requires_approval?)
+    unless stage.deploy_requires_approval?
       "Not Required"
-    elsif buddy.nil? && pending?
-      "Pending"
-    elsif buddy.nil?
-      "None"
-    elsif (user.id == buddy.id)
-      "Bypassed"
     else
-      buddy.name
+      if buddy.nil? && pending?
```

Comment:
this looks wrong ... should be `elsif` ... and that won't work work unless ... so
```Ruby
if !stage.deploy_requires_approval?
  ...
elsif ...
```